Elevel/Domain/Models/Test.cs

```diff
@@ -10,39 +10,51 @@ namespace Elevel.Domain.Models
     public class Test: BaseDataModel
     {
         public Guid UserId { get; set; }
-        public User User { get; set; }
+        //public ApplicationUser User { get; set; }
 
         public Guid? HrId { get; set; }
-        public User Hr { get; set; }
+        //public ApplicationUser Hr { get; set; }
 
         public Guid? CoachId { get; set; }
-        public User Coach { get; set; }
+
+        //public ApplicationUser Coach { get; set; }
 
         public Level Level { get; set; }
 
         public DateTimeOffset CreationDate { get; set; }
+
         public DateTimeOffset AssignmentStartDate { get; set; }
+
         public DateTimeOffset AssignmentEndDate { get; set; }
 
         public ICollection<Grammar> Grammars{ get; set; } = new List<Grammar>();
+
         public int? GrammarMark { get; set; }
 
         public Guid? AuditionId { get; set; }
+
         //public Audition Audition { get; set; }
+
         public int? AuditionMark { get; set; }
 
         public ICollection<UserAnswer> UserAnswers { get; set; } = new List<UserAnswer>();
 
         public Guid? EssayId{ get; set; }
+        
         //public Essay Essay { get; set; }
+
         public string EssayAnswer { get; set; }
+
         public int? EssayMark { get; set; }
 
         public Guid? SpeakingId { get; set; }
+
         //public Speaking Speaking { get; set; }
+
         public string SpeakingAnswerReference { get; set; }
-        public int? SpeakingMark { get; set; }
 
+        public int? SpeakingMark { get; set; }
+        
```

Comment:
Do you really need these spaces?
I made it that way for grouping it

Reply:
This spaces improve reading of code by my experience.